domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -1,32 +1,91 @@
 package org.oppia.domain.profile
 
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Matrix
+import android.media.ThumbnailUtils
 import android.net.Uri
+import android.provider.MediaStore
+import androidx.exifinterface.media.ExifInterface
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileDatabase
 import org.oppia.app.model.ProfileId
 import org.oppia.data.persistence.PersistentCacheStore
 import org.oppia.util.data.AsyncResult
+import org.oppia.util.data.DataProviders
 import org.oppia.util.logging.Logger
+import org.oppia.util.profile.DirectoryManagementUtil
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+import java.lang.Exception
+import java.math.BigInteger
+import java.nio.charset.Charset
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+import java.util.*
```

Comment:
Please import all classes we need rather than using * imports since those pollute the public namespace.

Reply:
Updated to have specific imports.